lib/server.js

```diff
@@ -55,7 +56,20 @@ process.on('unhandledRejection', (err) => {
     }
 });
 
+const options = {}; // (1/3) stupid gimmick because fuck the library lol
+options.definition = require('./swaggerDef.js'); // (2/3) cli options are slightly different
+options.apis = require('./swaggerDef.js').apis;
+// (3/3) so i have to make this gimmick
+const swaggerSpec = swaggerJSDoc(options); // Initialize swagger-jsdoc -> returns validated swagger spec in json format
+
 // Endpoints not requiring authorization.
+GeneralRouter.get('/api-docs.json', (req, res) => { // mini-route to retrieve the docs
```

Comment:
can we move it to `middlewares/` ? either to `/middlewares/generic.js` or create other file to store it

idea is that `lib/server.js` contains the routing (e.g. this route is served by these middlewares), and not the middlewares themselves (they are stored in `middlewares/*`